frontend/src/js/components/workspace/Workspaces.tsx

```diff
@@ -594,6 +598,27 @@ class WorkspacesUnconnected extends React.Component<Props, State> {
                     if (menuItemProps.content === 'Reprocess source file' && (isWorkspaceLeaf(entry.data))) {
                         this.props.reprocessBlob(workspaceId, entry.data.uri)
                     }
+
+                    if (menuItemProps.content === "Search in folder"){
+                        history.push(
+                            buildLink("/search", {
+                                q: JSON.stringify([
+                                    "",
+                                    {
+                                        n: "Workspace Folder",
+                                        v: entry.name,
+                                        op: "+",
+                                        t: "workspace_folder",
+                                        workspaceId: workspace.id,
+                                        folderId: entry.id,
+                                    },
+                                    "*"
```

Comment:
I was thinking this might have been a mistake but actually a good shout to start with * so that there are documents visible straight away before the user adds a search

Reply:
yep that's what I was thinking. Still want to test this in code with a workspace with loads of documents to see if it slows down the initial folder search.